task 5/server.js

```diff
@@ -0,0 +1,26 @@
+const http = require('http');
+const fs = require('fs');
+const socketIo = require('socket.io');
+
+const server = http.createServer((req, res) => {
+    res.writeHead(200, {
+
+    });
+    fs.readFile("./index.html", (err, fileContent) => {
+        res.end(fileContent);
+    });
+
+})
+
+const io = socketIo(server);
+
+io.on('connection', (socket) => {
+    socket.on('chat message', (msg) => {
+        io.emit('chat message', msg);
```

Comment:
To seperate events on backend and frontend, i suggest to emit another event on backend: such as `io.emit('serverData',...` and subscribe to it in fronend.